### PR TITLE
Pull Request chore/introduce-run-model

### PR DESCRIPTION
**Pull request by stravid**
This introduces the `Run` model on both client and server side.

### CODE REVIEW

File: app/models/run.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Run < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :track
```
Comment (DominikGuzei) [inline app/models/run.rb:3]: no time here?

**Comment by DominikGuzei** [comment]
Looks great -> mergable! :+1:

File: app/models/run.rb (hunk 1 above)
Comment (tombu) [inline app/models/run.rb:3]: @DominikGuzei: You usually don´t list the properties in rails.

File: spec/models/track_spec.rb [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 describe Track do
 
+  it { should validate_presence_of :rasterized }
+  it { should have_many(:runs) }
```
Comment (tombu) [inline spec/models/track_spec.rb]: please remove parentheses :wink: (or add them in the line above ... but i like the 'parentheses-less' style)

File: spec/models/run_spec.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+
+describe Run do
+  it { should belong_to(:user) }
+  it { should belong_to(:track) }
```
Comment (tombu) [inline spec/models/run_spec.rb]: please remove parentheses ...

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe User do
+  it { should have_many(:runs) }
```
Comment (tombu) [inline spec/models/user_spec.rb]: parentheses :wink:

File: app/models/run.rb (hunk 1 above)
Comment (DominikGuzei) [inline app/models/run.rb:3]: true  story :-D

**Pull request merged**